src/index.html

```diff
@@ -8,20 +8,400 @@
+        <p class="card__subtitle">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-label">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+            class="card__button"
+            href="#"
+            data-qa="hover"
+          >
+            BUY
+        </a>
+      </div>
+
+      <div
+          class="card"
+          data-qa="card"
+        >
+        <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+
+        <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__subtitle">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-label">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+            class="card__button"
+            href="#"
+            data-qa="hover"
+          >
+            BUY
+        </a>
+      </div>
+
+      <div
+          class="card"
+          data-qa="card"
+        >
+        <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+
+        <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__subtitle">Product code: 195434</p>
+        <div class="card__review">
+          <div class="stars stars--4">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__review-label">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-price">$2,199</span>
+        </div>
+        <a
+            class="card__button"
+            href="#"
+            data-qa="hover"
+          >
+            BUY
+        </a>
+      </div>
+    </main>
+  </body>
   </body>
```

Comment:
There is a duplicate `<body>` closing tag at line 406. Please remove this to ensure correct HTML structure.